gravitee-apim-portal-webui-next/src/app/api-details/api-details.component.ts

```diff
@@ -50,8 +51,14 @@ import { Api } from '../../entities/api/api';
 })
 export class ApiDetailsComponent implements OnInit {
   @Input() api!: Api;
+  isUserAuthenticated = false;
```

Comment:
You can just feed the signal `isUserAuthenticated` directly here. You can use the nav bar component as an example.

Reply:
Done